Plugins/LfpDisplayNode/LfpDisplayNode.cpp

```diff
@@ -329,6 +329,17 @@ void LfpDisplayNode::acknowledgeTrigger(int id)
     latestTrigger.set(id, -1);
 }
 
+bool LfpDisplayNode::getIntField(DynamicObject::Ptr payload, String name, int& value, int lowerBound, int upperBound) {
```

Comment:
I added this to Lfp for now but I think it would be nice to templatize this move this (and some other tools) into a Message Utility class when we can update the Plugin API.

Reply:
I incorporated this method into the Online PSTH plugin, and had to change one line to get it to compile on Linux: https://github.com/open-ephys-plugins/online-psth/commit/789ad31615b3dd814a28bd2def1bfbfd02a5ae27